tasks/agent6.yml

```diff
@@ -1,17 +1,31 @@
 ---
-- name: Create /etc/datadog-agent
+- name: Create Datadog agent config directory
   file:
     dest: /etc/datadog-agent
     state: directory
 
-- name: Create main Datadog agent yaml configuration file (beta)
+- name: Create main Datadog agent configuration file
   template:
     src: datadog.yaml.j2
     dest: /etc/datadog-agent/datadog.yaml
     owner: "{{ datadog_user }}"
     group: "{{ datadog_group }}"
   notify: restart datadog-agent
 
+- name: Ensure datadog-agent is running
```

Comment:
Shouldn't we first create every configuration files before starting the agent, or am I missing something ?

Reply:
hmmm let met test that ^^

Reply:
it works fine but I reverted to original order as this does not bring any profit and forces agent to (re)start 2 times instead only 1 at first install. let met know if you need other changes